src/client.js

```diff
@@ -1,12 +1,19 @@
 import Port from './port';
+import {clientSyncTitle} from './plugins/sync-title';
 
 export default class Client extends Port {
 	constructor(options) {
 
+		options = options || {};
+
 		super(window.parent, '*', options);
 
 		this.lastHeight = 0;
 
+		if(options.syncTitle === undefined || options.syncTitle) {
```

Comment:
could  just be `options.syncTitle !== false` (explicitly require `false` to disable it)